doc/source/release-notes/1.5.0.rst

```diff
@@ -0,0 +1,44 @@
+.. _PBM-1.5.0:
+
+================================================================================
+*Percona Backup for MongoDB* 1.5.0
+================================================================================
+
+:Date: May 10, 2021
+:Installation: `Installing Percona Backup for MongoDB <https://www.percona.com/doc/percona-backup-mongodb/installation.html>`_
+
+Percona Backup for MongoDB is a distributed, low-impact solution for consistent backups of MongoDB
+sharded clusters and replica sets. This is a tool for creating consistent backups
+across a MongoDB sharded cluster (or a single replica set), and for restoring
+those backups to a specific point in time. 
+
+.. important::
+
+   Backups made with previous versions are incompatible for restore with |PBM| 1.5.0. This is because processing of system collections ``users`` and ``roles`` has changed during backup / restore operations. For details, refer to :ref:`pbm.running.backup.restoring` and :jirabug:`PBM-636`. 
+
+New Features
+================================================================================
+
+* :jirabug:`PBM-596`: Azure Blob Storage support
+* :jirabug:`PBM-488`: Create weight or tag method to influence with ``pbm-agent`` node will do backups
+
+
+
+Improvements
+================================================================================
+
+* :jirabug:`PBM-662`: Show PITR Status based on ``admin.pbmLock`` instead of config settings
+* :jirabug:`PBM-494`: Prefer a (healthy) hidden secondary to any other node in automatic selection
+
+
+
+Bugs Fixed
+================================================================================
+
+* :jirabug:`PBM-642`: Display ``priority=0`` members on agent list in ``pbm status`` output
+* :jirabug:`PBM-636`: Different collection UUID after restore (Thanks to Nikolay for reporting this issue)
```

Comment:
should we mention also Dmitry Kuzmin (https://github.com/coraxster) for providing initial PR (https://github.com/percona/percona-backup-mongodb/pull/507)?

Reply:
Yes, we should. Thanks for remembering.